snuba/cleanup.py

```diff
@@ -31,7 +31,7 @@ def get_active_partitions(
 
     response = clickhouse.execute(
         """
-        SELECT DISTINCT partition
+        SELECT DISTINCT partition, partition_id
```

Comment:
Could you elaborate a bit on why you need the `partition_id` in the query?

Reply:
was a typo. removed.